microk8s-resources/actions/enable.portainer.sh

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why is the portainer addon removed @balasu ?

Reply:
Hi for this PR i would like only the traefik changes needs to be incorporated. Portainer changes i am still continuing